src/SwedbankPay.Sdk.Infrastructure/Payments/VippsPaymentsResource.cs

```diff
@@ -1,27 +1,28 @@
-﻿using System;
-using System.Net.Http;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.VippsPayments
-{
-    public class VippsPaymentsResource : ResourceBase, IVippsPaymentsResource
-    {
-        public VippsPaymentsResource(HttpClient httpClient) : base(httpClient)
-        {
-        }
-
-        public Task<VippsPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
-        {
-            if (id == null)
-                throw new ArgumentNullException(nameof(id));
-
-            return VippsPayment.Get(id, this.httpClient, GetExpandQueryString(paymentExpand));
-        }
-
-        public Task<VippsPayment> Create(VippsPaymentRequest paymentRequest,
-                                                            PaymentExpand paymentExpand = PaymentExpand.None)
-        {
-            return VippsPayment.Create(paymentRequest, this.httpClient, GetExpandQueryString(paymentExpand));
-        }
-    }
-}
+﻿using SwedbankPay.Sdk.Payments.VippsPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class VippsPaymentsResource : ResourceBase, IVippsPaymentsResource
+    {
+        public VippsPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<IVippsPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)